docs/openapi.yaml

```diff
@@ -225,11 +225,23 @@ paths:
               properties:
                 label:
                   type: string
-                action:
+                actions:
                   type: array
                   items:
                     type: object
                     properties:
+                      commands:
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            type:
+                              type: integer
+                            name:
+                              type: string
+                            parameters:
+                              type: array
+                              items: {integer}
```

Comment:
parameters can have various type, like string, number, object, ..., thus it would be better to not specify `items` so that `any` type can be given.

Reply:
I had that added as my Visual Studio Code previewer did not show anything when no `items` included. But it's removed.

Reply:
then `items: {}` should do the trick.